tests/API/OVAL/unittests/test_state_check_existence.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+result=`mktemp`
+
+set -e
+set -o pipefail
+
+cat > /tmp/foo.txt <<EOF
```

Comment:
I was in a similar situation in the past. In `test_remediation_simple.oval.xml` I used local file in the build directory for a similar purpose. What do you think about such solution?